pages/hydra-ecosystem-tools-overview.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+title: Tools | Hydraecosystem.org
+permalink: /hydra-ecosystem-tools
+---
+
+# Generics
+
+## Hydrus
+
+**Hydrus** is a generic server stack that is used for easier and efficient creation of Hypermedia driven REST-APIs with the use of Hydra. Hydrus utilizes the power of Linked Data to create powerful REST APIs to serve data. Hydrus uses the Hydra(W3C) standard for creation and documentation of it’s APIs. It is a project created from scratch with minimal requirements so that anyone can use it in their project without any difficulties. Hydra is basically a python based library, which enables us to create (smart) APIs which can be interpreted by machines. In order to leverage the benefit of Hydrus library apart from dependent python libraries, we don't need much apart from an API doc. Actually Hydrus expects hydraDoc object, but when we pass an API Doc to it. It internally uses the Doc Maker tool to create a HydraDoc object for it. After receiving hydraDoc object, Hydrus parses the data into `Classes`, `Properties`, `Collections`. Which are then utilized to create smart APIs.  
+
+Diagram shown below should make things more clear. 
+
+<img src="https://gsocchrizandr.files.wordpress.com/2017/06/flo1.png"/>
+
+Hydrus supports following features-
```

Comment:
This is from my initial proposal for hydrus. This is no longer valid. We have a separate agent and there is no middleware in hydrus.

Reply:
Again "Reference of this information". Any GSOC blog? where this update might have a mention?